site/content/blogs/release-v123.en.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'AWS Copilot v1.23: Try out App Runner private services and more!'
+twitter_title: 'AWS Copilot v1.23'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.23: Try out App Runner private services and more!
+
+Posted On: Oct 31, 2022
+
+The AWS Copilot core team is announcing the Copilot v1.23 release.   
+Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and has over 300 people online and nearly 2.5k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.23 brings several new features and improvements:
+
+- **App Runner private services**: [See detailed section](#app-runner-private-services).
+- **Additional source IPs to public ALB's security group**: [See detailed section](#additional-source-ips-to-public-albs-security-group).
+- **Deprecating environment manifest fields**[See detailed section](#deprecate-environment-manifest-fields).
```

Comment:
The title looks scary 😂! What do you think of something like
```
Moving misplaced `http` fields in environment manifest (backward-compatible!)
```

Reply:
yeah I had already changed it locally because I find it scary too after committing it. I will change it.